rafs/src/fs.rs

```diff
@@ -505,9 +508,8 @@ impl Rafs {
         device: BlobDevice,
     ) {
         // First do range based prefetch for rafs v6.
-        if sb.meta.is_v6() {
+        if sb.meta.is_v6() && !sb.superblock.get_blob_infos().is_empty() {
```

Comment:
let blob_infos = sb.superblock.get_blob_infos();
And it can reused in this file.

Reply:
Thanks

Reply:
fixed